geoapps_utils/base.py

```diff
@@ -252,7 +252,18 @@ def build(cls, input_data: InputFile | dict | None = None, **kwargs) -> Self:
 
         data.update(kwargs)
         options = Options.collect_input_from_dict(cls, data)  # type: ignore
-        out = cls(**options)
+
+        try:
+            out = cls(**options)
+        except ValidationError as errors:
+            summary = "\n - ".join(
+                f"{'.'.join(str(loc) for loc in error['loc'])}: {error['msg']}"
```

Comment:
```suggestion
                f"{'.'.join(str(loc) for loc in error['loc'])}: {error['msg']}  -> {error['input']}"
```
Can we add the input value? I think it will be useful in the long run to quickly debug issues with the file.